src/variable.cpp

```diff
@@ -81,6 +91,9 @@ std::string Variable::doSerialisation(FORMATS format) const
         if (getInitialValue().length()) {
             repr += " initial_value=\"" + getInitialValue() + "\"";
         }
+        if (getInterfaceType() != Variable::INTERFACE_TYPES::INTERFACE_TYPE_UNDEFINED) {
```

Comment:
I would simply test against ```INTERFACE_TYPE_NONE```. Even if the user explicitly specifies an interface of ```none```, I think it's fine not to have it in the serialisation.